ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_search.py

```diff
@@ -125,3 +126,41 @@ def test_paddle_nonzero(
         input=input[0],
         as_tuple=as_tuple,
     )
+
+
+# searchsorted
+@handle_frontend_test(
+    fn_tree="paddle.searchsorted",
+    dtype_and_values=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("valid"),
+        shared_dtype=True,
+        min_num_dims=1,
+        max_num_dims=1,
```

Comment:
`paddle.searchsorted` is supposed to work for 1-d or N-d inputs, so you shouldn't be setting `max_num_dims=1`. Other than that, great PR!

Reply:
got it, thanks!